kong/pdk/node.lua

```diff
@@ -0,0 +1,56 @@
+--- Node-level utilities
+--
+-- @module kong.node
+
+local utils = require "kong.tools.utils"
+
+
+local NODE_ID_KEY = "kong:node_id"
+
+
+local node_id
+
+
+local function new(self)
+  local _NODE = {}
+
+
+  ---
+  -- Returns the id used by this node to describe itself.
+  --
+  -- @function kong.node.get_id
+  -- @treturn string The v4 UUID used by this node as its id
+  -- @usage
+  -- local id, err = kong.node.get_id()
```

Comment:
The example says `get_id` may return a second `err` argument, but that's not the case. I suggest changing to `local node_id = assert(kong.node.get_id())` (no hard feelings either way about the assert, just an idea to convey "should always work")

Reply:
Ugh, a leftover from prior to this change! Thanks!